src/visit/reversed.rs

```diff
@@ -95,6 +95,16 @@ where
 #[derive(Copy, Clone, Debug)]
 pub struct ReversedEdgeReference<R>(R);
 
+impl<R> ReversedEdgeReference<R> {
+    /// Return the original, unreversed edge reference.
+    pub fn as_unreversed(&self) -> &R { &self.0 }
+
+    /// Consume `self` and return the original, unreversed edge reference.
+    pub fn into_unreversed(self) -> R {
```

Comment:
Nice in this case we know R: Copy because it's an edge ref, but no change is needed